### PR TITLE
SSP controls natsorted by id; polish LATO SSP output

### PR DESCRIPTION
**Pull request by gregelin**
Order of 800-53 controls natsorted to insure ordering of controls.

Polish Lightweight ATO SSP output:

* "What is the solution and how is it implemented" displayed only
when a control has implementation statements.
* Any control without any implementation statements has a message noted in red.
* HTML SSP page width set to max-width 950px to read better.

(Note: Page width and highlight of controls missing
coverage only show up currently in HTML SSP.)

### CODE REVIEW

**Comment by gregelin** [comment]
Commented out `controls.models.ElementControl.get_controls_by_element` because the method is not used in the code base. Will delete in future version. Closes #1446.

File: controls/models.py [hunk 1]
```diff
@@ -321,6 +321,9 @@ def selected_controls_oscal_ctl_ids(self):
         """Return array of selectecd controls oscal ids"""
         # oscal_ids = self.controls.all()
         oscal_ctl_ids = [control.oscal_ctl_id for control in self.controls.all()]
+        # Sort
+        oscal_ctl_ids = natsorted(oscal_ctl_ids, key=str.casefold)
```
Comment (davidpofo) [inline controls/models.py:325]: nice use

Comment (gregelin) [reply controls/models.py:325]: Thanks!

**Comment by gregelin** [comment]
@davidpofo Can you add review - approved, pls?

**Review by davidpofo — APPROVED** [review]

**Pull request merged**